src-plugins/itkFilters/itkMorphologicalFiltersProcessBase.cpp

```diff
@@ -70,6 +72,13 @@ void itkMorphologicalFiltersProcessBase::setParameter(double data, int channel)
     }
 }
 
+void itkMorphologicalFiltersProcessBase::setParameter(int data)
+{
+    DTK_D(itkMorphologicalFiltersProcessBase);
```

Comment:
This class uses the DTK approach to private implementation, and this macro is needed in all functions that want to use the d pointer.